Examples/Algorithms/TrackFindingML/src/AmbiguityResolutionMLAlgorithm.cpp

```diff
@@ -8,18 +8,29 @@
 
 #include "ActsExamples/TrackFindingML/AmbiguityResolutionMLAlgorithm.hpp"
 
+#include "ActsExamples/EventData/IndexSourceLink.hpp"
+#include "ActsExamples/EventData/Measurement.hpp"
 #include "ActsExamples/Framework/ProcessCode.hpp"
 
 #include <iterator>
 #include <map>
 
+std::size_t sourceLinkHash(const Acts::SourceLink& a) {
```

Comment:
If this is used only here, then let's go for internal linkage
```suggestion
static std::size_t sourceLinkHash(const Acts::SourceLink& a) {
```

Reply:
I usually do this with an anonymous namespace.